### PR TITLE
NoTask- Endpoint for project copy

### PR DESCRIPTION
**Pull request by lgrzybowski**
New endpoint for copy project.

### CODE REVIEW

**Comment by benSlaughter** [comment]
Rather than a new file, looking at the existing project endpoints, it looks like this should be in the projects file.
take a look at existing endpoints [here](https://github.com/jonathanchrisp/bwapi/blob/master/lib/bwapi/client/projects.rb#L91)

**Comment by lgrzybowski** [comment]
@benSlaughter Thx for comment, fixed.

**Comment by lgrzybowski** [comment]
Is this ready to merge ? @benSlaughter @jonathanchrisp

**Comment by leemcgeever** [comment]
the checks failed - is that expected?  aside from that looks good :)

**Comment by lgrzybowski** [comment]
ha @leemcgeever welcome :) 
Yes it's looks like problem with installing `oj` gem for ruby in head version `An error occurred while installing oj (2.13.0), and Bundler cannot continue.`
For version 2.1.0, 2.0.0 and super old 1.9.3 works fine

**Comment by leemcgeever** [comment]
I wonder if that is compatible with the latest version?  Anyway,  I'll stop interfering 😄

**Comment by jonathanchrisp** [comment]
@lgrzybowski tests are failing so please can you look into this?

File: lib/bwapi/client/projects.rb [hunk 1]
```diff
@@ -91,6 +91,16 @@ def project_audiences(project_id, opts = {})
         get "/projects/#{project_id}/audiences", opts
       end
 
+      # Copy Published Projects into current user account
+      #
+      # @param project_id [Integer] Id of project
+      # @param opts [Hash] options hash of parameters
+      # @option opts [Hash] structureOnly should copy only the structure (no backfills)
+      # @return [Hash] Copied project
+      def copy(project_id, opts = {})
```
Comment (jonathanchrisp) [inline lib/bwapi/client/projects.rb]: can we make this more descriptive e.g. `copy_project`

**Comment by lgrzybowski** [comment]
@jonathanchrisp I made a change of method name. 
The problem is not with tests but with build gem on head ruby version. I think it's not related to my change:
```
An error occurred while installing oj (2.13.0), and Bundler cannot continue.
Make sure that `gem install oj -v '2.13.0'` succeeds before bundling.
```

**Comment by jonathanchrisp** [comment]
Looks good following the latest merge from @benSlaughter, merging :+1: Please test the pre release gem and let me know how it looks.

**Pull request merged**